packages/core/src/components/button/_button-group.scss

```diff
@@ -147,7 +146,6 @@ Styleguide components.button-group.css
 
       &::after {
         @include pt-button-minimal-divider();
-
```

Comment:
It is banned with the new `declaration-empty-line-before` rule and I couldn't figure out a clean way to enforce our own style (I want to enforce no blank lines between plain rule declarations). I don't think this whitespace is helpful; syntax highlighting of the `@ignore` token works pretty great IMO